pages/releases/4.3.html

```diff
@@ -0,0 +1,2313 @@
+								</h4>
+								<div class="c-content">
+									<p>
+										In this release, we introduced a new standardized system for XR hand tracking,
+										which allows developers to use one kind of node (and code) to create a hand
+										tracking experience that will work in multiple XR eco-systems. This is currently
+										supported by both OpenXR and WebXR.
+									</p>
+									<p>
+										Brand new features are systems for body and face tracking, that are built in the
+										same standartized way, but currently only supported in OpenXR on Meta Quest headsets.
+									</p>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #xr-standardized-hand-body-face-tracking -->
+
+				<div id="xr-performance-stability-improvements" class="release-card span-3">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#xr-performance-stability-improvements">
+										Performance and stability improvements
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										Due to countless improvements made to this part of the engine, XR applications are now more stable and performant.
+									</p>
+									<p>
+										Highlights include:
+									</p>
+									<ul>
+										<li>
+											improved timing of tracking data
+										</li>
+										<li>
+											XR is now thread safe, allowing you to move rendering onto a separate thread
+										</li>
+										<li>
+											foveated rendering has been tweaked on the <em>Mobile</em> and <em>Forward+</em>
+											renderers
+										</li>
+										<li>
+											glasses-based AR & passthrough-based AR have been unified
+										</li>
+									</ul>
+								</div>
+								<div class="c-link"
+									data-contributors="BastiaanOlij,dsnopek"
+									data-read-more="https://github.com/godotengine/godot/pulls?q=87630+89734+89894">
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #xr-performance-stability-improvements -->
+
+				<div id="xr-meta-xr-simulator-support" class="release-card span media-bottom">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#xr-meta-xr-simulator-support">
+										Meta XR Simulator support
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										One complication of developing XR games is having to repeatedly test your
+										application on a headset, especially if your headset is a standalone VR
+										unit (read: not connected to a computer).
+									</p>
+									<p>
+										The Meta XR Simulator is a free tool to test your XR application outside
+										of the headset, but only works if you are targeting the Quest series models.
```

Comment:
The "only works if your are targeting the Quest series models" doesn't seem quite right. The Meta XR Simulator is simulating a Quest, but you absolutely can use it as a time saver even if you're targeting a different headset altogether. I expect lots of people will do that, given that there aren't a lot of these simulators around.